kiss

```diff
@@ -743,22 +743,16 @@ pkg_checksums() {
     # Generate checksums for packages.
     repo_dir=$(pkg_find "$1")
 
-    # Support packages without sources. Simply do nothing.
-    [ -f "$repo_dir/sources" ] || return 0
-
     while read -r src dest || [ "$src" ]; do
-        # Skip comments, blank lines and git sources.
-        if [ -z "${src##\#*}" ] || [ -z "${src##git+*}" ]; then
+        # Skip directories comments, blank lines and git sources.
```

Comment:
'directories, comments, blank lines, and git sources' ?

Reply:
Oops, missed a comma. Thanks :+1: